src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -122,4 +125,51 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String date} into a {@code LocalDate}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code date} is invalid.
+     */
+    public static LocalDate parseDate(String date) throws ParseException {
```

Comment:
Ok. I suggest for now we leave it as is. Since Lesson and Task have a date component, we can standardise once both have been merged in and refactor it to be one of the 'commons'. Then that will be cleaner. Maybe leave for v1.3

Reply:
Okay then maybe you create an issue for that?